<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsbPro</class>
 <widget class="QWidget" name="UsbPro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UsbPro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(238, 255, 250, 200);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>失败</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;测试步骤&lt;/span&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;A. 拔掉pc与夹具 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri'; font-size:10.5pt;&quot;&gt;usb&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;数据线，连接 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri'; font-size:10.5pt;&quot;&gt;OTG &lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;数据线及 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri'; font-size:10.5pt;&quot;&gt;U&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;盘；&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;B. 待测 PCBA自动读取 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri'; font-size:10.5pt;&quot;&gt;U&lt;/span&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;盘数据并显示；&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; font-weight:600;&quot;&gt;判定：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt;&quot;&gt;写入一个文件，然后读取文件，对比ok即为通过，否则失败&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>读取U盘</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; font-weight:600;&quot;&gt;测试步骤：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;A. 拔掉pc与夹具 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;usb&lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;数据线，连接 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;OTG &lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;数据线及 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;U&lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;盘；&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;B. 待测 PCBA自动读取 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;U&lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;盘数据并显示；&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体'; font-size:10.5pt; font-weight:600;&quot;&gt;判定：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;可以正常读取 U盘数据则手动判定为 &lt;/span&gt;&lt;span style=&quot; font-family:'宋体','Calibri';&quot;&gt;Pass&lt;/span&gt;&lt;span style=&quot; font-family:'宋体';&quot;&gt;；否则为Fail。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	image: url(:/ProductTest_360/Tip.jpg);
border-style: outset;
border-width: 4px;
border-radius: 10px;
border-color: rgba(255, 170, 0, 30);
}
QLabel:hover{
background-color:rgba(100,255,100, 100);
border-color: rgba(255, 170, 0, 255);
color:rgba(0, 0, 0, 255);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProductTest_360.qrc"/>
 </resources>
 <connections/>
</ui>
